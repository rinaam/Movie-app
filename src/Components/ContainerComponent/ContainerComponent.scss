.wrapper {
  margin: 0 auto;
  max-width: 1200px;
  padding: 4px;
}
