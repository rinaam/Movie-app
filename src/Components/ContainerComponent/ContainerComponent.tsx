import React from "react";
import "./ContainerComponent.scss";

const ContainerComponent: React.FC = ({ children }) => (
  <div className="wrapper">{children}</div>
);

export default ContainerComponent;
