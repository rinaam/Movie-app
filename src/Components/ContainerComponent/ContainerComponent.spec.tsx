import { create } from "react-test-renderer";
import ContainerComponent from "./ContainerComponent";
import React from "react";

describe("ContainerComponent", () => {
  it("should render ContainerComponent", () => {
    const wrapper = create(
      <ContainerComponent>Hello World</ContainerComponent>
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
