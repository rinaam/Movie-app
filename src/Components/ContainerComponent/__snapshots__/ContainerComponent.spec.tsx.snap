// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerComponent should render ContainerComponent 1`] = `
<div
  className="wrapper"
>
  Hello World
</div>
`;
